### PR TITLE
React practice

### PR DESCRIPTION
**Pull request by jerryowusu**
*Please remove all italic texts including this one. Feel free to add/remove text as per your need.*

## Description

Learn and practice the fundamentals of React from start to intermediate

### General requirements

*Check items below by adding x between brackets.*

- [x] There are **no linter errors**.
- [x] Followed the **GitFlow**.
- [x] The documentation is **professional**.

### HTML/CSS requirements

*Feel free to remove the item(s) not related to this PR.*

- [x] Followed best practices for JavaScript

### Project requirements

- [x] Form and event handling
- [x] Implementing CSS in React app
- [x] Create, add, edit and delete
- [x] Persisting data in local storage
- [x] Routing with react Router
- [x] Deploying React to Github pages

### Additional notes

Feel free to add any additional notes here if needed or remove it entirely

---
Thank you for taking the time to review this pull request.

### CODE REVIEW

File: src/classBased/components/TodoContainer.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+class TodoContainer extends React.Component {
+    state = {
+        todos: []
+       };
+
+    //    componentDidMount() {
+    //        fetch("https://jsonplaceholder.typicode.com/todos?_limit=10")
+    //        .then(response => response.json)
+    //        .then(data => this.setState({todos: data}));
+    //    }
```
Comment (mwafrika) [inline src/classBased/components/TodoContainer.js:17]: [OPTIONAL] @jerryowusu  please remove the commented codes to make your code much cleaner

File: src/classBased/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <header style={headerStyle}>
+        <h1
+            style={{
+                fontSize: "6rem",
+                fontWeight: "600",
+                marginBottom: "2rem",
+                lineHeight: "1em",
+                color: "#ececec",
+                textTransform: "lowercase",
+                textAlign: "center",
```
Comment (mwafrika) [inline src/classBased/components/Header.js:18]: [OPTIONAL] Kindly, move your CSS to a specific class everywhere in your project. remember that you can only put all your styling in a separate file and then import it in your code to use its rules. it's a good practice to improve performance and to clean up your code 😢

**Review by mwafrika — APPROVED** [review]
Wow. well done @jerryowusu ,
Your project is complete, there is nothing other than saying it's time to merge your pull request
# 💯 🥇 🍏

while your project is perfect, you can find below the optional work for your project to keep it cleaner.

**Comment by jerryowusu** [comment]
Thank you @mwafrika

File: src/classBased/components/TodoContainer.js (hunk 1 above)
Comment (jerryowusu) [reply src/classBased/components/TodoContainer.js:17]: Thank you @mwafrika

File: src/classBased/components/Header.js (hunk 2 above)
Comment (jerryowusu) [reply src/classBased/components/Header.js:18]: Thanks for the feedback @mwafrika. I will put that into consideration

**Pull request merged**